docs/standard/terminology.md

```diff
@@ -0,0 +1,69 @@
+# Terminology
+
+In order to use key terms consistently across the standard documentation and OCP communication in general, terminology is managed according to a process that ensures both quality and reactivity.
+
+![Terminology process overview](https://www.lucidchart.com/invitations/accept/2eee45ed-5e62-4636-9a44-24b4414f273f)
+
+## 1. Proposition and validation
```

Comment:
I would suggest calling this stage 'Proposal and review'

Proposition can have a couple of other meanings, and we use the technical term validation elsewhere, so want to avoid confusing by using it here.

Reply:
OK